Beis.LearningPlatform.Web/wwwroot/js/site.js

```diff
@@ -94,4 +94,30 @@ for (i = 0; i < acc.length; i++) {
             panel.style.display = "block";
         }
     });
-}
\ No newline at end of file
+}
+
+//cms_component_NextPageRadioButton
+$(document).ready(function () {
+    $("#cms_component_NextPageRadioButton_divjsenabled").attr("style", "");
+});
+$('input[type=radio][name=cms_component_NextPageRadioButton_rdon]').change(function () {
```

Comment:
We dont use jQuery - please remove

Reply:
DONE in last commit
Changed to pure javascript